SourceMock/Actions/VeinSource/VeinSource.cs

```diff
@@ -28,12 +28,20 @@ public SourceCapabilities GetCapabilities()
 
         public Loadpoint? GetCurrentLoadpoint()
         {
-            throw new NotImplementedException();
+            Loadpoint ret = new();
+
+            var veinResponse = _veinClient.GetLoadpoint();
+            // how to act on http statuscode and pass through to api endpoint?
```

Comment:
Maybe the status codes (+ error description?) can be mapped to a `SourceResult`?